_drafts/2021-01-08-ALTF4-20.md

```diff
@@ -0,0 +1,78 @@
+
+Even though regional differences might seem like the biggest troublemaker, what kills us are actually the more subtle quirks, like the fact that we have a formal and a more informal version for 'you', and those sometimes completely change the structure of a sentence. In some settings, it's unambiguous on which form you should use, but especially for a blog, there are great arguments for both sides. (This is also the reason why I don't like dubbing, even big budget films get it wrong all the time, and it just feels soo unnatural when someone "Siezt" a close friend.)
+
+One thing where it pays off to have different people working on the same thing though is synonyms. Or technically 'not-synonyms', because even though some words might have a similar meaning to most people, especially in tech some words sound a bit off, even though they translate to the same english word as the correct one. For scenarios like this it really pays off to have your translation proofread by up to 4 more people, all with other areas of expertise. However this is also were our biggest weekly challenge comes in:
+
+### Heading
+
+Translating takes time. We don't have that. Currently, articles come in at Thursday morning, we start translating at noon or in the afternoon, and have a first translation in the evening. But then someone wants to proofread it. And I can't remember the last time, we didn't have corrections after someone went through a translation. Corrections might not always be better though, even though they were well-meant[?] (we have a German word for that, "verschlimmbessern", *to worsen something by trying to improve it*); thus we now always have the original translator read through the corrections. But you guessed it, that takes time too. If we have a good day, and everyone is available, we are sometimes less than half an hour before release still busy ironing out some hiccups, correcting errors, and fiddling with non-synonyms.
+
+Sometimes these problems, especially the more technical ones, result in in changes for all languages, sometimes even english [maybe reference if touched upon]. For example quotes are where they are now, because we couldn't figure out how to incorporate translations while still keeping the original quote. [I feel like there should be something more, but I don't know what]
+
+### Heading
```

Comment:
same title as the one before

Reply:
This may need a more direct refering to how to translate factorio article (F4-9):
"Multiple translation processus"